sehaj-deep/Warzone_Game
# Refactor Logger Using Observer Pattern

To test these changes: 
1. Start the game and play normally.
2. Observe the src>main>resources>logs.txt file changing when you use a command.
